### PR TITLE
Rework usages, users, webhooks, and website descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks usages, users, webhooks, and website descriptions to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/paths/users@{id}@totp-reset.yaml [hunk 1]
```diff
@@ -5,13 +5,13 @@ post:
     - Users
   tags:
     - Users
-  summary: Reset (renew) totpSecret
+  summary: Reset or renew a totpSecret
```
Comment (ashkarpetin) [inline openapi/paths/users@{id}@totp-reset.yaml:8]:
```suggestion
  summary: Reset or renew a TOTP secret
```

File: openapi/paths/users@{id}@totp-reset.yaml [hunk 2]
```diff
@@ -5,13 +5,13 @@ post:
...
   operationId: PostUserTotpReset
   x-sdk-operation-name: resetTotp
-  description: Reset (renew) `totpSecret`.
+  description: Resets and renews a totpSecret with a specified ID.
```
Comment (ashkarpetin) [inline openapi/paths/users@{id}@totp-reset.yaml:11]:
```suggestion
  description: Resets and renews a TOTP secret with a specified user ID.
```

Comment (noon-dawg) [reply openapi/paths/users@{id}@totp-reset.yaml:11]: Thanks @ashkarpetin!

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/Usage.yaml [hunk 3]
```diff
@@ -40,7 +40,9 @@ properties:
     minimum: 0
     exclusiveMinimum: true
   usageTime:
-    description: Date and time, in ISO 8601 format, at which a usage occurred. If this value is not provided or is empty, the date and time of the request is used.
+    description: |-
+      Date and time, in ISO 8601 format, at which a usage occurred.
```
Comment (adamaltman) [inline openapi/components/schemas/Usage.yaml:44]:
```suggestion
      Date and time, in ISO 8601 format, when a usage occurred.
```

**Review by nanobash — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**